src/dev-app/clipboard/clipboard-demo-module.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ClipboardModule} from '@angular/cdk/clipboard';
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {RouterModule} from '@angular/router';
+import {ClipboardDemo} from './clipboard-demo';
+
+@NgModule({
+  imports: [
+    CommonModule,
```

Comment:
Nit: Is the common module needed here?

Reply:
It's not. I'm a little surprised that Bazel didn't complain about it since it's not in the build file.

Reply:
Yeah, that's interesting. I don't think we add `@npm//@angular/common` as default dependency.